assertk-common/src/main/kotlin/assertk/failure.kt

```diff
@@ -58,23 +61,29 @@ internal class SoftFailure : Failure {
 
     override fun invoke() {
         if (!failures.isEmpty()) {
-            fail(compositeErrorMessage(failures))
+            FailureContext.failure.fail(compositeErrorMessage(failures))
         }
     }
 
-    private fun compositeErrorMessage(errors: List<AssertionError>): String {
+    private fun compositeErrorMessage(errors: List<AssertionError>): AssertionError {
         return if (errors.size == 1) {
-            errors.first().message.orEmpty()
+            errors.first()
         } else {
-            errors.joinToString(
-                    prefix = "The following ${errors.size} assertions failed:\n",
-                    transform = { "- ${it.message}" },
-                    separator = "\n"
-            )
+            AssertkMultipleFailuresError(errors)
         }
     }
 }
 
+// Override message with our previous behavior.
+internal class AssertkMultipleFailuresError(private val errors: List<Throwable>) :
```

Comment:
Is this a good idea? Will it be a problem for any tooling? Or is it worth changing the output to match the opentest4j default.

Reply:
https://github.com/ota4j-team/opentest4j/issues/46